src/GPL-2.0-only.xml

```diff
@@ -27,8 +27,11 @@
       </p>
     </standardLicenseHeader>
     <notes>
-      This license was released: June 1991 This refers to when this
-      GPL 2.0 only is being used (as opposed to GPLv2 or later).
+      This license was released: June 1991. This license identifier refers to when 
+      GPL-2.0-only is being used as opposed to GPL-2.0-or-later. The option to use either that
```

Comment:
Can I suggest replacing "The option to use... " with "The license header of individual files will indicate which of 'only' or 'later' applies to that particular file." I _think_ that's the same intent, but if not, might want to clarify in a different way?